build_projects/dotnet-cli-build/CompileTargets.cs

```diff
@@ -270,9 +274,30 @@ private static void RemovePdbsFromDir(string directory)
                 GenerateNuGetPackagesArchive(c, dotnet, sdkOutputDirectory);
             }
 
+            CopyMSBuildTargetsToSDKRoot(sdkOutputDirectory);
+
             return c.Success();
         }
 
+        private static void CopyMSBuildTargetsToSDKRoot(string sdkOutputDirectory)
+        {
+            var msbuildTargetsDirectory = Path.Combine(sdkOutputDirectory, "runtimes", "any", "native");
```

Comment:
I don't know, this is the way many packages are authored in the cli ecosystem currently. 

I think it's something we should fix though